sonic-thermalctld/scripts/thermalctld

```diff
@@ -761,7 +761,6 @@ class ThermalControlDaemon(daemon_base.DaemonBase):
             self.log_warning('Thermal manager is not supported on this platform')
         except Exception as e:
             self.log_error('Caught exception while initializing thermal manager - {}'.format(repr(e)))
-            sys.exit(ERR_INIT_FAILED)
```

Comment:
Since this is the only place `ERR_INIT_FAILED` is used, you should also remove the definition at the top of the file.

Reply:
Sure